docs/user-stories.md

```diff
@@ -0,0 +1,9 @@
+## Glottie Chat User Stories
+
+1. As a researcher whose native language is not English, I want to communicate with peer colleagues in a chat-room where all messages are translated automatically, so that I don't have to fight with English grammar and correct word usage.
+2. As a chat service user, I want to obtain a unique account, so that my personal identity couldn't be forged.
+3. As an authenticated user, I need to be able to create new chat-rooms dedicated to cerain topics and invite other fellow researchers to join them.
+4. As a chat participant, I want to preview the content of each of my messages and make sure the system makes an adequate semantic interpretation of it, so that I could rest assured the intended meaning will come across.
+5. As an invited chat guest, I want to join the discussions without registration, so that I could remain anonymous.
+6. As a chat participant, I want to see all messages from other users translated to my mother tongue, so that I don't have to learn other foreign languages.
+7. As a researcher, I need to communicate with people from other countries, so that we could form an international research community.
```

Comment:
Doesn't look like a user story for the chat app, but it looks great as a motivation part for 3rd user story :)

Reply:
I admit, I'm a newbie in Agile methodology, will try and refactor this one. =)